packages/suite/src/views/wallet/coinmarket/common/CoinmarketHeader/CoinmarketHeaderFilter.tsx

```diff
@@ -35,6 +37,14 @@ const InputWrapper = styled.div`
 const CoinmarketHeaderFilter = () => {
     const context = useCoinmarketOffersContext<CoinmarketTradeBuySellType>();
```

Comment:
```suggestion
    const context = useCoinmarketOffersContext();
```

Please, delete the type.